docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+services:
+  ubuntu_node:
+    build: .
+    ports:
+      - "2224:22"
+
+  # This is the same as used in the Vagrantfile
+  centos_node:
+    image: jdeathe/centos-ssh:centos-6
+    ports:
+      - "2226:22"
```

Comment:
Is this for tests that haven't been written? Doesn't look like anything interacts with 2 targets yet.

Reply:
Yes, I added it because I wasn't sure which os we should use, and didn't see too much harm in it being there? But I also don't have specific plans for it, so can remove it.